gulpfile.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+var gulp = require("gulp");
```

Comment:
Looks good.  Just wondering why you chose to write it in gulp rather than adding to the webpack config?

Reply:
I tried looking for a way to do it through the webpack config but wasn't able to - I think webpack isn't set up to export two copies of the same file, I was only able to figure out how to compile an scss file to css or to export as is, but not to do both with the same file.

Reply:
Ah, yeah. It's way more complicated to get webpack to do that than it should be!  I'm going to approve your approach but also create a ticket to convert this to webpack ; just in case anybody on the team has a specific knack for webpack configuration (and/or the time to work on it).